test/rowHeight.test.ts

```diff
@@ -21,37 +20,24 @@ function setupGrid(rowHeight: DataGridProps<Row>['rowHeight']) {
   setup({ columns, rows, rowHeight });
 }
 
-test('rowHeight is number', async () => {
+test('rowHeight is number', () => {
   setupGrid(40);
 
   const grid = screen.getByRole('grid');
-  expect(grid).toHaveStyle({ 'grid-template-rows': 'repeat(1, 40px) repeat(50, 40px)' });
-  expect(getRows()).toHaveLength(31);
-
-  await userEvent.tab();
-  expect(grid.scrollTop).toBe(0);
-
-  // Go to the last cell
-  const spy = vi.spyOn(window.HTMLElement.prototype, 'scrollIntoView');
-  await userEvent.keyboard('{Control>}{end}');
-  expect(spy).toHaveBeenCalled();
+  expect(grid).toHaveStyle({
+    'grid-template-rows':
+      '40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px 40px'
```

Comment:
Why does it not use the `repeat()` syntax? Is it because the "resolved" value is different than the input?

Reply:
I think it is because we are using `toHaveStyle`. `expect(grid.style.gridTemplateRows).toBe('repeat(1, 40px) repeat(50, 50px)');` works fine.

Reply:
https://github.com/testing-library/jest-dom/blob/main/src/to-have-style.js#L60

Reply:
Maybe we can reduce the amount of rows we render then 🙂